presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -2641,7 +2641,62 @@ public void testSelectCaseInsensitive()
     @Test
     public void testShowSession()
     {
-        Session session = new Session(
+        Session session = buildSession();
+        MaterializedResult result = computeActual(session, "SHOW SESSION");
+
+        ImmutableMap<String, MaterializedRow> properties = Maps.uniqueIndex(result.getMaterializedRows(), input -> {
+            assertEquals(input.getFieldCount(), 5);
+            return (String) input.getField(0);
+        });
+
+        assertEquals(properties.get("test_string"), new MaterializedRow(1, "test_string", "foo string", "test default", "varchar", "test string property"));
```

Comment:
These should all be able to go in dedicate tests in the engine (i.e., in `io.prestosql.sql.query`)